sdk/src/wallet/operations/syncing/transactions.rs

```diff
@@ -202,14 +198,15 @@ where
         }
         drop(wallet_data);
 
-        for mut transaction in transactions_to_reissue {
-            log::debug!("[SYNC] reissue transaction");
-            let reissued_block = self
-                .submit_signed_transaction(transaction.payload.clone(), None)
-                .await?;
-            transaction.block_id.replace(reissued_block);
-            updated_transactions.push(transaction);
-        }
+        // TODO: this is a problem
```

Comment:
I guess we shouldn't do this automatically for now (or never), since it also requires more mana which doesn't get allotted if we just submit the same payload

Reply:
Should I just remove all the related code then?

Reply:
I think we should do it in an own PR, because it affects more than what you commented